README.md

```diff
@@ -39,10 +45,91 @@ point in having those in this fork for various reasons.
+|:-------------------------|:-------------------------|:---------------------------------------------------------------------------------------------|----------------------------------|
+| ZLIB_COMPAT              | --zlib-compat            | Compile with zlib compatible API                                                             | OFF                              |
+| ZLIB_ENABLE_TESTS        |                          | Build test binaries                                                                          | ON                               |
+| WITH_GZFILEOP            | --with-gzfileops         | Compile with support for gzFile related functions                                            | OFF                              |
+| WITH_MSAN                | --with-msan              | Build with memory sanitizer                                                                  | OFF                              |
+| WITH_OPTIM               | --without-optimizations  | Build with optimisations                                                                     | ON                               |
+| WITH_NEW_STRATEGIES      | --without-new-strategies | Use new strategies                                                                           | ON                               |
+| WITH_NATIVE_INSTRUCTIONS |                          | Instruct the compiler to use the full instruction set on this host (gcc/clang -march=native) | OFF                              |
+|                          | --force-sse2             | Assume SSE2 instructions are always available                                                | DISABLED (x86), ENABLED (x86_64) |
+| WITH_ACLE                | --without-acle           | Build with ACLE CRC                                                                          | ON                               |
+| WITH_NEON                | --without-neon           | Build with NEON intrinsics                                                                   | ON                               |
+| WITH_DFLTCC_DEFLATE      | --with-dfltcc-deflate    | Use DEFLATE COMPRESSION CALL instruction for compression on IBM Z                            | OFF                              |
+| WITH_DFLTCC_INFLATE      | --with-dfltcc-inflate    | Use DEFLATE COMPRESSION CALL instruction for decompression on IBM Z                          | OFF                              |
+| WITH_SANITIZERS          | --with-sanitizers        | Build with address sanitizer and all supported sanitizers other than memory sanitizer        | OFF                              |
+| WITH_FUZZERS             | --with-fuzzers           | Build test/fuzz                                                                              | OFF                              |
+
+Install
+-------
+
+We do not recommend installing unless you know what you are doing, as this can
```

Comment:
I suggest the following text instead, making the warning more clear and a little easier to spot:

WARNING: We do not recommend manually installing unless you really know what you are doing,
because this can potentially override the system default zlib library, and any incompatability or wrong
configuration of zlib-ng can make the whole system unusable, requiring recovery or reinstall.
If you still want a manual install, we recommend using the /opt/ path prefix.

For Linux distros, an alternative way to use zlib-ng (if compiled in zlib-compat mode) instead of zlib, is through
the use of the _LD_PRELOAD_ environment variable. If the program is dynamically linked
with zlib, then zlib-ng will temporarily be used instead by the program, without risking system-wide instability.

Reply:
Changed.